src/snowflake/snowpark/modin/pandas/resample.py

```diff
@@ -241,19 +241,34 @@ def ffill(self, limit: Optional[int] = None) -> Union[pd.DataFrame, pd.Series]:
         )
 
     def backfill(self, limit: Optional[int] = None):
-        self._method_not_implemented("backfill")  # pragma: no cover
+        return self.bfill(limit=limit)
 
-    def bfill(self, limit: Optional[int] = None):  # pragma: no cover
-        self._method_not_implemented("bfill")
+    def bfill(self, limit: Optional[int] = None):
+        is_series = not self._dataframe._is_dataframe
+
+        if limit is not None:
+            ErrorMessage.not_implemented(
+                "Parameter limit of resample.bfill has not been implemented."
+            )
+
+        return self._dataframe.__constructor__(
+            query_compiler=self._query_compiler.resample(
+                self.resample_kwargs,
+                "bfill",
+                (),
+                {},
+                is_series,
+            )
+        )
 
-    def pad(self, limit: Optional[int] = None):  # pragma: no cover
-        self._method_not_implemented("pad")
+    def pad(self, limit: Optional[int] = None):
```

Comment:
Do we also need to update changelog and docs to reflect that `pad` is also partially supported?

Reply:
No `pad` is the same as `ffill`, but only applies as a parameter to `fillna` and not a top-level `resample` method.

Reply:
got it, thanks!